City-of-Pori/pori-kada
# PORI-172 Sections

To test:

1. drush fra -y && drush cc all
2. Create a section and make sure all fields on it work.
3. Add some pages for the section navigation (add pages to the section and its navigation)
4. Make sure the navigation works nicely and that the subpages content works correctly
5. Add some news and flag it for current flow and add it under the section and make sure it comes up in the current flow under the section.
